std_msgs/Header header
NodeSpline base_linear
NodeSpline base_angular
NodeSpline[] ee_motion
NodeSpline[] ee_force
PhaseDurations[] phase_durations
bool ee_durations_change